src/main/target/UAVPNG030MINI/target.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "UAVP"
+
+#define USBD_PRODUCT_STRING "UAVP-NG HW-0.30-mini"
+
+#define LED0_PIN                PE5
+#define LED1_PIN                PE7
+#define LED2_PIN                PE6
+#define LED3_PIN                PE8
+
+#define BEEPER                  PB0
+#define BEEPER_INVERTED
+
+#define ACC
+#define USE_ACC_SPI_MPU6000
+
+#define GYRO
+#define USE_GYRO_SPI_MPU6000
+
+#define GYRO_1_CS_PIN          PA4
+#define GYRO_1_SPI_INSTANCE    SPI1
+
+// TODO
+#define GYRO_MPU6000_ALIGN       CW180_DEG
+#define ACC_MPU6000_ALIGN        CW180_DEG
+
+// MPU6000 interrupts
+#define USE_EXTI
+#define MPU_INT_EXTI             PE0
+#define USE_MPU_DATA_READY_SIGNAL
+
+#define MAG
+#define USE_MAG_HMC5883
+#define MAG_HMC5883_ALIGN        CW90_DEG
+#define MAG_I2C_INSTANCE         I2CDEV_2
+#define HMC5883_I2C_INSTANCE     I2CDEV_2
+#define MAG_INT_EXTI             PE12
+#define USE_MAG_DATA_READY_SIGNAL
+
+#define BARO
+#define USE_BARO_MS5611
```

Comment:
Hardware page https://ng.uavp.ch/Family#HW-0.30-mini lists MS5607 as a barometer. Is the part software compatible with MS5611?

Reply:
I found this:
'MS5607 is temperature and pressure sensor which hardware is similar to MS5611.
Both sensors share command protocol and support both I2C and SPI serial
protocols. They only differ in compensation algorithms.'